heeji/가까운-수.js

```diff
@@ -0,0 +1,8 @@
+function solution(array, n) {
+    const newarr = [];
+    for(let item = 0; item < array.length; item++) {
+        newarr.push(Math.abs(array[item] - n));
+    }
+    const result = array.sort((a, b) => a - b);
+    return result.find(el => Math.abs(n - el) === Math.min(...newarr));
+}
```

Comment:
```suggestion
function solution(array, n) {
    const newarr = [];
    for(let i = 0; i < array.length; i++) {
        newarr.push(Math.abs(array[i] - n));
    }
    const result = array.sort((a, b) => a - b);
    return result.find(el => Math.abs(n - el) === Math.min(...newarr));
}
```
- 풀이 잘 봤습니다 👍
- for문에서는 보통 i, j, k 를 사용합니다~!